builtin/aws/ecs/platform.go

```diff
@@ -34,6 +36,42 @@ func (p *Platform) Config() (interface{}, error) {
 	return &p.config, nil
 }
 
+// ConfigSet is called after a configuration has been decoded
+// we can use this to validate the config
+func (p *Platform) ConfigSet(config interface{}) error {
```

Comment:
Is this function called automatically through the SDK or something internal?

Reply:
Yup! It's something the SDK supports, we just didn't use it before in ECS.